### DISCUSSION TITLE
github application reject password whereas desktop version signin ok

### DISCUSSION THREAD

**Discussion by pilere** [community/community#6357] [Mobile]
Hi,
I don't get why the mobile application of github reject my signin with a bad password message whereas the desktop allow me to get in .... what's wrong ? How can I debug this ?

**Comment by brianlovin** [comment]
Hey @pilere sorry for the trouble here — have you managed to get into the app since posting this?

**Reply by pilere** [reply]
sorry, I forgot and since changed a lot my usage ....

**Comment by Bali0531** [comment]
same for me.
When I try to login to my account on pc it says its okey man yeah... its correct
But when i try on my phone (browser or Github app) it says nahh its incorrect. I tried to change it and the same happend.
